content/_index.html

```diff
@@ -91,14 +91,14 @@ <h3 class="mt-5 has-text-white">Requirements</h3>
             <code>nvidia-smi</code>
 
             <h3 class="mt-6 has-text-white">Install with Conda</h3>
-            <p><strong>1.</strong> If not installed, download and run the install script. <br> 
-             This will install the latest miniconda: 
+            <p><strong>1.</strong> If not installed, download and run the install script. <br>
+             This will install the latest miniconda:
             </p>
             <pre class="highlight use-white-copy"><code>wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
 bash Miniconda3-latest-Linux-x86_64.sh</code></pre>
             <br>
             <p><strong>2.</strong> Then quick install RAPIDS with:</p>
-            <pre class="highlight use-white-copy"><code>conda create -n rapids-23.06 -c rapidsai -c conda-forge -c nvidia rapids=23.06 python=3.10 cudatoolkit=11.8</code></pre>
+            <pre class="highlight use-white-copy"><code>conda create -n rapids-23.08 -c rapidsai -c conda-forge -c nvidia rapids=23.08 python=3.10 cudatoolkit=11.8</code></pre>
 
             <h3 class="mt-6 has-text-white">Install with pip</h3>
             <p><strong>1.</strong> Install via pip channels. <br> Not all RAPIDS libraries are currently available:</p>
```

Comment:
```suggestion
            <p><strong>1.</strong> Install via pip channels.</p>
```
With 23.08 all RAPIDS standard libraries are on pip (excluding cuSignal because of its pending removal)